app/src/features/NavigationBreadcrumbs/NavigationBreadcrumbs.tsx

```diff
@@ -56,9 +57,23 @@ const useStyles = makeStyles((theme) => ({
     justifyContent: "center",
     alignItems: "center",
   },
+  breadcrumbsContainer: {
+    display: "flex",
+    padding: `0 ${theme.spacing(5)}px`,
+    minHeight: theme.mixins.breadcrumbBar.height,
+    alignItems: "center",
+  },
+  children: { marginLeft: "auto" },
 }));
 
-const NavigationBreadcrumbs = (): JSX.Element => {
+type NavigationBreadcrumbsProps = MuiBreadcrumbsProps & {
+  editMappingButton?: JSX.Element;
```

Comment:
maybe we could find a better/generic name

Reply:
Navbar ?